server/xpub-model/entities/auditLog/index.js

```diff
@@ -12,7 +12,7 @@ class AuditLog extends BaseModel {
         userId: { type: 'uuid' }, // Currently not used
         action: {
           type: 'string',
-          enum: ['CREATED', 'UPDATED', 'DELETED', 'MECA_RESULT'],
+          enum: ['CREATED', 'UPDATED', 'DELETED', 'MECA_RESULT', 'APPLICATION'],
```

Comment:
You probably committed this by mistake - this is to do with the AuditLog changes